tests/test_blockchain_gateway.py

```diff
@@ -0,0 +1,61 @@
+import tests.context	
+import pytest	
+from core.configuration import ConfigurationManager	
+from core.blockchain.blockchain_gateway import BlockchainGateway
+from core.utils.enums import RawEventTypes
+from core.blockchain.tx_utils import TxEnum
+@pytest.fixture	
```

Comment:
Please add two empty lines above this line.

Reply:
done